fastfilter/src/main/java/org/fastfilter/bloom/BlockedBloom.java

```diff
@@ -50,7 +50,7 @@ public void add(long key) {
         long m1 = (1L << hash) | (1L << (hash >> 6));
         long m2 = (1L << (hash >> 12)) | (1L << (hash >> 18));
         data[start] |= m1;
-        data[start + 1 + (int) (hash >>> 60)] |= m2;
```

Comment:
This would change the false positive rate I think... Not sure how much, but probably measurable. Instead, the array should be made larger.